lib/zipkin-tracer/config.rb

```diff
@@ -8,7 +8,8 @@ class Config
     attr_reader :service_name, :service_port, :json_api_host,
```

Comment:
I mean is there a way to make a config spec that prevents this sort of bug from happening again? I don't know the answer to this